CMakeLists.txt

```diff
@@ -360,8 +358,18 @@ add_library(wabt STATIC ${WABT_LIBRARY_SRC})
 
 IF (NOT WIN32)
   add_library(wasm-rt-impl STATIC wasm2c/wasm-rt-impl.c wasm2c/wasm-rt-impl.h)
-  install(TARGETS wasm-rt-impl DESTINATION ${CMAKE_INSTALL_LIBDIR})
-  install(FILES wasm2c/wasm-rt.h wasm2c/wasm-rt-impl.h DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
+  if (WABT_INSTALL_RULES)
+    install(
+      TARGETS wasm-rt-impl
+      COMPONENT WABT_Development
+    )
+    install(
+      DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/wasm2c/"
+      TYPE INCLUDE
+      COMPONENT WABT_Development
+      FILES_MATCHING PATTERN "wasm-rt*.h"
```

Comment:
Can we keep this as a list of two files for now, just to be explicit, and also the keep this change more focused?

We can change it a wildcard later perhaps if the list of headers ever grows?  (I doubt it will).

Reply:
Sure, happy to back that out a bit.

Reply:
Done.